next/components/forms/simple-components/ServiceCard.tsx

```diff
@@ -0,0 +1,49 @@
+import cx from 'classnames'
+import { FunctionComponent, HTMLAttributeAnchorTarget, SVGProps } from 'react'
+
+import CircleArrowRight from '../../../assets/images/circle-arrow-right.svg'
+
+type ServiceCardBase = {
+  title: string
+  description: string
+  buttonText: string
+  Logo: FunctionComponent<SVGProps<SVGSVGElement>>
+  href: string
+  target?: HTMLAttributeAnchorTarget | undefined
+  className?: string
+}
+
+const ServiceCard = ({
+  title,
+  description,
+  buttonText,
+  Logo,
+  href,
+  target,
+  className,
+}: ServiceCardBase) => {
+  const style = cx(
+    'w-[280px] bg-gray-0 border-gray-200 flex flex-col items-start p-5 gap-5 border-solid border-2 rounded-lg',
+    className,
+  )
+
+  return (
+    <a href={href} target={target} rel="noreferrer">
+      <div role="button" tabIndex={0} className={style} aria-hidden="true">
+        <div className="min-w-[75px] min-h-[75px] max-w-[75px] max-h-[75px] flex items-center justify-center">
```

Comment:
once again - you are using a lot of classes which are not really needed, I think you can get pretty much the same behaviour here with just:

```suggestion
        <div className="w-[75px] h-[75px]">
```